Civi/Api4/MembershipLog.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+namespace Civi\Api4;
+
+/**
+ * MembershipLog entity.
+ *
+ * @searchable primary
+ * @since 5.42
```

Comment:
This is definitely not a `primary` entity. Careful with copy-paste!
Also it's not since 5.42. Master is on 5.64 now.